assets/loadtest/k8s/Makefile

```
@@ -431,41 +431,56 @@ run-tc-scaling-test:
 	kubectl wait --for=condition=available deploy/tc -n $(NAMESPACE) --timeout=60m
 	@sleep 120
 	@make setup-tc
-	
+
 	@sleep 300
-	
+
 	@make delete-tc
 
 # collect goroutine and heap go profiles from the auth deployment
 .PHONY: collect-profiles
 collect-profiles: collect-auth collect-proxy collect-node
 
+# forwards the pods diagnostics address
+.PHONY: forward-pod-diagnostics
+forward-pod-diagnostics:
+	kubectl port-forward $(POD) 3434:3434 -n $(NAMESPACE) > /dev/null 2>&1 &
+
+# retrieves profiles from all pods matching the selector
+.PHONY: collect-pods
+collect-pods:
+	@for POD in $$(kubectl get pod -n $(NAMESPACE) -l $(SELECTOR) -o custom-columns=name:metadata.name --no-headers) ; do \
+make collect-pod POD=$$POD ; \
```

nit: stylistic

```suggestion
$(MAKE) collect-pod POD=$$POD ; \
```

same for others.